src/ng1/directives/treeView/treeView.partial.html

```diff
@@ -1,4 +1,4 @@
-<div class="tree-node tree-node-content" data-nodrag ng-click="tv.select(this)" ng-class="{highlight : tv.isSelected(this)}" >
+<div class="tree-node tree-node-content {{node.class}}" data-nodrag ng-click="tv.select(this)" ng-class="{highlight : tv.isSelected(this)}" >
```

Comment:
Change this to something like `className` rather that `class` as it is a registered javascript keyword.